pages/store/[nftClassId]/index.vue

```diff
@@ -587,6 +622,39 @@ const checkoutButtonProps = computed<{
   }
 })
 
+const recommendedClassIds = computed(() => {
+  let items: string[] = []
+  if (bookInfo.bookstoreInfo.value?.recommendedClassIds) {
+    items = items.concat(bookInfo.bookstoreInfo.value.recommendedClassIds)
+  }
+  const ownedClassIds = authorStore.getOwnedClass(bookInfo.nftClassOwnerWalletAddress.value).map(item => item.classId)
+  return items.concat(ownedClassIds).filter(id => id !== nftClassId.value).slice(0, 10)
+})
+
+const recommendedItemInfos = computed(() => {
+  return recommendedClassIds.value.map((classId) => {
+    return {
+      classId,
+      ...(nftStore.getNFTClassMetadataById(classId) || {}),
+    }
+  })
+})
+
+watch(() => recommendedClassIds.value, (newItems) => {
+  if (newItems.length) {
+    newItems.forEach((classId) => {
+      nftStore.lazyFetchNFTClassAggregatedMetadataById(classId).catch(() => {
+        console.warn(`Failed to fetch aggregated metadata for the NFT class [${classId}]`)
+      })
+    })
+  }
+})
```

Comment:
Since this will be called in `<BookstoreItem/>`, this can be removed?
https://github.com/likecoin/3ook-com/blob/88e1a1e3d4e8fbe5a5fe082427f86173d937420f/components/BookstoreItem.vue#L84-L108